ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionGroupResult.java

```diff
@@ -36,7 +36,7 @@
 })
 /*
  The original deserialised private transaction sent via eea_sendRawTransaction
- This class is used if the original request was sent with privateFrom and privateFor
+ This class is used if the original request was sent with privacyGroupId
```

Comment:
It'd be good to update the comment in PrivateTransactionLegacyResult as well (same reason).

Reply:
This has now been addressed. Sorry for the delay.